source/code/plugin/health/health_model_definition_parser.rb

```diff
@@ -0,0 +1,50 @@
+=begin
+    Class to parse the health model definition. The definition expresses the relationship between monitors, how to roll up to an aggregate monitor,
+    and what labels to "pass on" to the parent monitor
+=end
+require 'json'
+
+module HealthModel
+    class HealthModelDefinitionParser
+        attr_accessor :health_model_definition_path, :health_model_definition
+
+        # Constructor
+        def initialize(path)
+            @health_model_definition = {}
+            @health_model_definition_path = path
+        end
+
+        # Parse the health model definition file and build the model roll-up hierarchy
+        def parse_file
+            if (!File.exist?(@health_model_definition_path))
+                raise "File does not exist in the specified path"
```

Comment:
ensure this is handled and impacts only health

Reply:
Yes. Verified that.